dev-tools/mage/build.go

```diff
@@ -88,8 +91,16 @@ func DefaultBuildArgs() BuildArgs {
 	}
 
 	if FIPSBuild {
-		args.ExtraFlags = append(args.ExtraFlags, "-tags=requirefips")
-		args.CGO = true
+
+		fipsConfig := packaging.Settings().FIPS
+
+		for _, tag := range fipsConfig.Compile.Tags {
+			args.ExtraFlags = append(args.ExtraFlags, "-tags="+tag)
```

Comment:
Would this send `-tags=A -tags=B`?
Tags need to be sent as a comma seperated list otherwise they are silently ignored (send `-tags=A,B`)

Reply:
This should already be taken care of by https://github.com/elastic/elastic-agent/blob/58bcc269c18f5887ca9d10b6fca3efe120408b45/dev-tools/mage/build.go#L48-L71
Introduced with PR https://github.com/elastic/elastic-agent/pull/6565

Reply:
hmmm that replies in one of my in-progress comments 🙂